zmq.hpp

```diff
@@ -630,6 +632,27 @@ inline void swap(message_t &a, message_t &b) ZMQ_NOTHROW
     a.swap(b);
 }
 
+#ifdef ZMQ_CPP11
+enum class ctxopt
+{
+    blocky = ZMQ_BLOCKY,
+    io_threads = ZMQ_IO_THREADS,
+    thread_sched_policy = ZMQ_THREAD_SCHED_POLICY,
+    thread_priority = ZMQ_THREAD_PRIORITY,
+    thread_affinity_cpu_add = ZMQ_THREAD_AFFINITY_CPU_ADD,
+    thread_affinity_cpu_remove = ZMQ_THREAD_AFFINITY_CPU_REMOVE,
+    thread_name_prefix = ZMQ_THREAD_NAME_PREFIX,
+    max_msgsz = ZMQ_MAX_MSGSZ,
+#ifdef ZMQ_BUILD_DRAFT_API
```

Comment:
Hm, this should probably say 
```suggestion
#ifdef ZMQ_ZERO_COPY_RECV
```
since not every (older) version of libzmq has it, even when `ZMQ_BUILD_DRAFT_API` is defined.